Sample/Kitchen Sink/DemonstrationTools/CrossWindCalculator.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Shape+Extensions.swift
+//  flight-ui-ios
+//
+//  Created by Appivate 2023
+//
+
+import SwiftUI
+import Foundation
+import FlightUI
+
+struct CrossWindCalculator: View {
+
+    @StateObject var viewModel = CrosswindCalculatorViewModel()
+    @EnvironmentObject var theme: Theme
+
+    var body: some View {
+        ScrollView {
+            VStack {
+                HStack() {
+                    runwayNumber
+                    windSpeedInput
+                    windDirection
+                }
+                windspeedOutput
+                    .padding([.top],theme.padding.grid8x)
+            }
+            .padding(theme.padding.grid2x)
+            .navigationBarTitle("Crosswind Calculator")
+        }
+    }
+
+    var runwayNumber: some View {
+        MenuField(selection: $viewModel.runwayNumber,
+                  options: Array(1...36),
```

Comment:
Comment: I didn't even think about the ability to do this as an Array, great idea 😂 🎉